docs/reference/admin/layouts/_Example/components/Directives.tsx

```diff
@@ -1,18 +1,19 @@
-import { createDirectiveContext } from '@contember/layout';
-import * as React from 'react';
+import { Directives } from '@contember/layout';
 
-type DirectivesType = {
-  'title': string | null | undefined;
+export type DirectivesType = {
+  'full-width': boolean
 }
 
-const directivesDefaultValues: DirectivesType = Object.freeze({
-  'title': undefined,
+export const initialDirectives: DirectivesType = Object.freeze({
```

Comment:
omit the Object.freeze (cannot directives provider do the freeze?)

Reply:
No longer needed, removed.